l10n_br_nfe/models/document.py

```diff
@@ -239,6 +239,9 @@ def _compute_dest(self):
     nfe40_infCpl = fields.Char(
         compute='_compute_nfe40_additional_data',
     )
+    nfe40_transporta = fields.Many2one(
+        comodel_name='res.partner'
+    )
```

Comment:
isso a principio nao deveria precisar, vamos checar esse ponto melhor.

Reply:
Ok. I'll separate in 2 commits, if it's not necessary you can just cherry pick the right commit.

Reply:
I tested the code locally and this configuration is still necessary.

Reply:
Pessoal esse commit é necessário. Se não a cada NF-e é necessário digitar todos os dados do transportador em vez de pegar do cadastro de clientes.

Reply:
Integrei ele no #1300